### PR TITLE
Inbox API check assets are ready

### PR DESCRIPTION
**Pull request by Sergio-Mira**

### CODE REVIEW

**Comment by shanethmoore** [comment]
Looks good :rocket: if you could just answer my question on the coding style in https://github.com/Swrve/swrve-sdk/pull/1091

File: SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java [hunk 1]
```diff
@@ -1830,11 +1829,12 @@ public C getConfig() {
...
-                if (campaign.isInbox() && campaign.getStatus() != SwrveCampaignState.Status.Deleted && campaign.isActive(getNow())) {
-
-                    if (campaign.supportsOrientation(orientation)) {
-                        result.add(campaign);
-                    }
+                if (campaign.isInbox()
+                        && campaign.getStatus() != SwrveCampaignState.Status.Deleted
+                        && campaign.isActive(getNow())
+                        && campaign.supportsOrientation(orientation)
+                        && campaign.areAssetsReady()) {
```
Comment (dominicmarmionswrve) [inline SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java:1836]: break this out into a seperate method

Comment (dominicmarmionswrve) [inline SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java:1836]: As in the boolean part of the if statement. Easier to test this way

File: SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java [hunk 2]
```diff
@@ -1830,11 +1829,12 @@ public C getConfig() {
         synchronized (campaigns) {
```
Comment (dominicmarmionswrve) [inline SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java:1829]: I know this isn't your fault, but the fact we're calling this method getCampaigns, which only returns Inbox campiagns, bugs me! 

@oisin Any update on this?

**Pull request merged**